app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -485,7 +660,15 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_configChange_lastMonthSectionTitleIsDisplayed() {
+  fun testRecentlyPlayedTestActivity_configChange_lastMonthSectionTitleIsDisplayed() { // ktlint-disable max-line-length
```

Comment:
/ ktlint-disable max-line-length not required

Reply:
Removed.